src/main/scala/pureconfig/WordDelimiter.scala

```diff
@@ -0,0 +1,42 @@
+package pureconfig
+
+trait WordDelimiter {
+  def toTokens(s: String): Seq[String]
```

Comment:
Ok, it would be more verbose as you'd have to extract all fields of the target class individually rather than provide per-field "exceptions". But now that you suggested that, I think I would prefer to have a custom `ConfigConvert` rather than add a custom field conversion implicit :)